platforms/js/build_js.py

```diff
@@ -239,6 +245,8 @@ def build_loader(self):
     # Write a path to modify file like argument of this flag
     parser.add_argument('--config', default=os.path.join(os.path.dirname(os.path.abspath(__file__)), 'opencv_js.config.py'),
                         help="Specify configuration file with own list of exported into JS functions")
+    parser.add_argument('--opencv_contrib_dir', default='',
+                        help="Path to /opencv_contrib/modules folder. Can be used to build Aruco marker support into opencvjs.")
```

Comment:
Not needed. Users should specify it by `--cmake_option="-DOPENCV_EXTRA_MODULES_PATH=opencv_contrib/modules"`

Reply:
Let's add this to the doc instead: https://docs.opencv.org/master/d4/da1/tutorial_js_setup.html

Reply:
I updated the documentation and removed the argument from the parser.